blocks/multiplatform/choose-share/blocks/where/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { Fragment, useState } from 'react';
+import cn from 'classnames';
+import { useMS } from '@jetbrains/kotlin-web-site-ui/out/components/breakpoints-v2';
+import { useTextStyles } from '@rescui/typography';
+import { Chip, ChipList } from '@rescui/chip-list';
+import { AndroidIcon, AppleIcon, ComputerIcon, GlobusIcon, ServerIcon } from '@rescui/icons';
+
+import styles from './choose-share-where.module.css';
+
+import { useTabScroll } from '../../hooks/useTabScroll';
+
+
+const TABS_BLOCKS = [
+    {
+        id: 'android',
+        tab: 'Android',
+        Icon: props => <AndroidIcon {...props} />,
+        Content: () => (
+            <>
+                Easily make your Android app cross-platform while keeping the majority of your Kotlin code. Many popular
+                libraries and architecture components&nbsp;– including those from Google, such as Room, DataStore, or
+                ViewModel&nbsp;– can be used with Kotlin&nbsp;Multiplatform.
+            </>
+        )
+    },
+    {
+        id: 'ios',
+        tab: 'iOS',
+        Icon: props => <AppleIcon {...props} />,
+        Content: () => (
+            <>
+                Share business logic seamlessly. Make your Kotlin functions available to call from Swift code, and use
+                the full power of iOS APIs in your Kotlin code. Use Compose Multiplatform to create screens that can be
```

Comment:
There are a few more mentions of "Compose Multiplatform" in other texts, probably they also should be rendered as links